arrow/factory.py

```diff
@@ -192,6 +197,11 @@ def get(self, *args, **kwargs):
             elif isinstance(arg, struct_time):
                 return self.type.utcfromtimestamp(calendar.timegm(arg))
 
+            # (iso calendar) -> convert then from date
+            elif isinstance(arg, tuple):
```

Comment:
I am a bit worried that `tuple` is too broad. I wonder if there is any other datetime information we can absorb in the `get()` method that would be of type `tuple`.